src/main/java/com/example/bookstore/controller/BookController.java

```diff
@@ -44,24 +46,27 @@ public BookDto getBookById(@PathVariable Long id) {
         return bookService.findById(id);
```

Comment:
a general recommendation, add @PreAuthorize("hasRole('USER')") to the relevant endpoints to avoid misunderstandings in the future

Reply:
Agree